src/components/Piece.tsx

```diff
@@ -0,0 +1,17 @@
+import styles from "../css/piece.module.css";
+
+interface PieceProps {
+    piece: string;
+    color: string;
+}
+
+export const Piece = (props: PieceProps) => {
+    if (!props.piece) return <></>;
+    return (
+        <i
+            className={`${styles.piece} fas fa-chess-${props.piece} ${
+                props.color === "white" ? styles.white : styles.black
+            }`}

```

Comment:
Not crucial, but you may be interested in this for the future: https://github.com/JedWatson/classnames